internal/provider/resource_managed_identity_access_rules.go

```diff
@@ -265,6 +278,9 @@ func (t *managedIdentityAccessRuleResource) Create(ctx context.Context,
 	accessRule.Type = types.StringValue(string(created.Type))
 	accessRule.RunStage = types.StringValue(string(created.RunStage))
 	accessRule.ManagedIdentityID = types.StringValue(created.ManagedIdentityID)
+	if !accessRule.VerifyStateLineage.IsNull() {
```

Comment:
The `VerifyStateLineage` field is marked as computed so this should be setting the value regardless of whether it was passed in.